cli/azd/pkg/apphost/manifest.go

```diff
@@ -217,6 +227,12 @@ func ManifestFromAppHost(
 				*res.Context = filepath.Join(manifestDir, *res.Context)
 			}
 		}
+
+		if res.BindMounts != nil {
+			for _, bindMount := range res.BindMounts {
+				bindMount.Source = filepath.Clean(filepath.Join(manifestDir, bindMount.Source))
```

Comment:
nit: redundant -- filepath.Join does `Clean` on the result:

> Join joins any number of path elements into a single path, separating them with an OS specific [Separator](https://pkg.go.dev/path/filepath#Separator). Empty elements are ignored. The result is Cleaned.

https://pkg.go.dev/path/filepath#Join